lib/editor/components/EntityListButtons.js

```diff
@@ -142,7 +142,8 @@ export default class EntityListButtons extends Component<Props, State> {
             <Button
               bsSize='small'
               data-test-id={`clone-${activeComponent}-button`}
-              disabled={!activeEntity}
+              // prohibit user from creating additional duplicates when there are unsaved entities
+              disabled={!activeEntity || entities.findIndex(entityIsNew) !== -1}
```

Comment:
Bit of a nitpick: you can probably use `!!entities.find(entityIsNew)` instead and save yourself the comparison!